images/nfs/Makefile

```diff
@@ -19,7 +19,7 @@ include ../image.mk
 
 NFS_IMAGE = $(BUILD_REGISTRY)/nfs-$(GOARCH)
 
-NFS_BASE ?= fedora:28
+NFS_BASE ?= ubuntu:xenial
```

Comment:
what caused the base image to change from fedora to ubuntu?

Reply:
ahh, I tried with fedora image but I was not able to build the image as some packages were not getting installed. Seems like some packages got deprecated or maybe a network problem at that time.